docs/AtasDeReunião/Ata_17-06.md

```diff
@@ -0,0 +1,29 @@
+# Ata de reunião 17/06
+
+Reunião realizada no Discord
+
+Data da reunião: 17 de Junho de 2022
+
+Redigida por: Ana Carolina
+
+Participantes:
+- [Amanda Nobre](https://github.com/AmandaNbr)
+- [Ana Carolina](https://github.com/AnaCarolinaRodriguesLeite)
+- [Irwin Schmitt](https://github.com/irwinschmitt)
+- [Lucas Braun](https://github.com/lbvx)
+- [Mateus Maia](https://github.com/mateusmaiamaia)
+- [Matheus Calixto](https://github.com/matheuscvp)
+- [Matheus Fonseca](https://github.com/gatotabaco) 
+- [Matheus Monteiro](https://github.com/matheusyanmonteiro)
+- [Thaís Rebouças](https://github.com/thais-ra)
+- [Yudi Yamane](https://github.com/yudi-azvd)
+
+## Pauta da reunião
+
+A equipe optou por fazer essa técnica no remoto onde foi separado uma reunião longa onde cada um dos integrantes tinham acesso ao miro (quadro branco online) e ao discord (plataforma de comunicação) possibilitando que todos tenham voz e a organização.
+
+## Reunião
+
+### Primeiro momento
+No início da reunião, a participante [Amanda Nobre](https://github.com/AmandaNbr) criou um quadro com vários postits em que cada era para ser preenchido por todo os membros, colocando suas ideias de que o projeto deve conter, fazer, especificar, aplicar, mostrar ou realizar da forma como cada membro acha que seria importante ter no projeto.
```

Comment:
No início da reunião, a participante [Amanda Nobre](https://github.com/AmandaNbr) criou um quadro com vários postits de modo que cada um dos membros da reunião preencheu colocando suas ideias para requisitos funcionais e não funcionais.